org.kde.koko.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "org.kde.koko",
+    "rename-icon": "koko",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "koko",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--env=XDG_DATA_DIRS=/usr/share:/app/share/"
```

Comment:
why do you need to set this? it's already set in the sandbox.

Reply:
Good question, I copied it from the nightly kde flatpak builds and it seems to be needed for correct rendering of thumbnails. @JBBgameich do you remember why it is needed, you added it with https://invent.kde.org/packaging/flatpak-kde-applications/-/merge_requests/17/diffs?

Reply:
It was needed for /app/share/kservices5/imagethumbnail.desktop to be found. But if the variable is now set globally in the runtime, it can of course be removed.